lib/modules/platform/github/massage-markdown-links.ts

```diff
@@ -58,12 +68,18 @@ export function massageMarkdownLinks(content: string): string {
   try {
     const rightSpaces = content.replace(content.trimEnd(), '');
     const matches: UrlMatch[] = [];
-    remark().use(collectLinkPosition(content, matches)).processSync(content);
+    const newContent = content.replace(
+      regEx(/@&#8203;(?<name>[A-Za-z1-9]*)/gi),
```

Comment:
regex should match everything that is not space 
example name: `@philip-abed` will not be caught here.